docs/src/upgrade/release-3.2.x-incubating.asciidoc

```diff
@@ -88,6 +88,11 @@ to the list of locally computed clauses.
 
 See: link:https://issues.apache.org/jira/browse/TINKERPOP-1764[TINKERPOP-1764]
 
+Clone a graph
+^^^^^^^^^^^^^
+If you want to quickly clone a graph (e.g. because you have set it up in a test and your traversals manipulate the graph) you can now use `GraphHelper.cloneElements(Graph original, Graph clone)`. 
```

Comment:
Please mention that this new class is in `gremlin-test` - "...you can now use `GraphHelper.cloneElements(Graph original, Graph clone)` which is part of the `gremlin-test` module."